### PR TITLE
Implement INI ConfigFormatter

### LINKED ISSUE #452: Implement `INI` ConfigFormatter
### Suggestion

introduce an `INI` file config formatter. 

### Motivation

Enhancing the flexibility of our formatters suite and catering to users of `INI` files.

### Context

- Install the [ini](https://www.npmjs.com/package/ini) package
- Refer to this [PR](https://github.com/configu/configu/pull/253/files) as a previous formatter implementations. 

Example usage:
```bash
configu eval ... | configu export --format 'INI'
```

### PR DESCRIPTION
**Pull request by michaldziuba03**
- installed `ini` package from npm
- used #253 as reference

Closes #452

### CODE REVIEW

**Comment by davidmeirlevy** [comment]
Thanks for your contribution!
LGTM.

@michaldziuba03 is it a draft or ready to use?
can you add a test?

File: packages/common/src/index.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 // import './test';
 
-export * from './Registry';
+export * from './registry';
```
Comment (michaldziuba03) [inline packages/common/src/index.ts:3]: it is not related to this PR but idk - without that change I am unable to build project at all

File: packages/lib/src/formatters/formatters.test.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        KEY1: {
+          type: 'String',
+        },
+      }),
+      configs: {
+        KEY0: 'KEY0',
+        KEY1: 'KEY1',
+      },
+    }).run();
+
```
Comment (michaldziuba03) [inline packages/lib/src/formatters/formatters.test.ts:24]: I made some tests for basic formats.
I decided to put tests inside `lib` dir where tested feature "lives" - I see most tests are located inside `ts` dir though. 

I don't see any existing test cases for other formatters, so I guess I was supposed to create them.

**Comment by michaldziuba03** [comment]
@davidmeirlevy I marked PR to be ready to review.

File: packages/lib/src/formatters/formatters.test.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  test('should export to yaml format', () => {
+    const formatted = formatConfigs({ json, format: 'YAML', label: 'test' });
+    equal(formatted, 'KEY0: KEY0\nKEY1: KEY1\n');
+  });
+
+  test('should export to json format', () => {
+    const formatted = formatConfigs({ json, format: 'JSON', label: 'test' });
+    equal(formatted, '{\n  "KEY0": "KEY0",\n  "KEY1": "KEY1"\n}');
+  });
+});
```
Comment (michaldziuba03) [inline packages/lib/src/formatters/formatters.test.ts:30]: I made some tests for basic formats.
I decided to put tests inside `lib` dir where tested feature "lives" - I see most tests are located inside `ts` dir though. 

I don't see any existing test cases for other formatters, so I guess I was supposed to create them?

File: packages/common/src/index.ts (hunk 1 above)
Comment (rannn505) [reply packages/common/src/index.ts:3]: @michaldziuba03 thanks for noticing, @jamesL92 just fixed it at #473, so we go ahead to merge this cool new formatter :)

File: packages/lib/src/formatters/formatters.test.ts (hunk 3 above)
Comment (rannn505) [reply packages/lib/src/formatters/formatters.test.ts:30]: @michaldziuba03 Thanks for adding tests for the new formatter! `Configu` has been lacking in this area, and we plan to invest heavily in drastically increasing our test coverage.

**Review by rannn505 — APPROVED** [review]
@michaldziuba03 Your work on this PR is nothing short of amazing! Thanks for such a solid and mature contribution. 🙌

**Pull request merged**